JASP-Engine/JASP/R/bainttestbayesianindependentsamples.R

```diff
@@ -238,9 +238,8 @@ BainTTestBayesianIndependentSamples <- function(jaspResults, dataset, options, .
 
 	  descriptivesTable$addColumnInfo(name="v",                    title = "", type="string")
 		descriptivesTable$addColumnInfo(name="group",                title = "Group", type="string")
-	  descriptivesTable$addColumnInfo(name="N",                    title = "N", type="integer")
+	  descriptivesTable$addColumnInfo(name="N",                    title = "n", type="integer")
```

Comment:
We call this `N` in other places (as per https://blog.apastyle.org/apastyle/2011/08/the-grammar-of-mathematics-writing-about-variables.html)